### PR TITLE
PAL-1088 updated performance test readme, and updated profiles in cre…

### PR DESCRIPTION
**Pull request by developer01189998819991197253**
…atePerformanceData and deployServicesToK8s

### CODE REVIEW

File: deployment-jvm/local-jvm/performance/createPerformanceData.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@
 FILE=performance/target/performance-*-exec.jar
```
Comment (m78233) [inline deployment-jvm/local-jvm/performance/createPerformanceData.sh:16]: Minor note, I've seen the use of JARFILE which seems like a better description of what it is.

**Review by m78233 — COMMENTED** [review]
Just the one comment

File: deployment-jvm/local-jvm/performance/createPerformanceData.sh (hunk 1 above)
Comment (developer01189998819991197253) [reply deployment-jvm/local-jvm/performance/createPerformanceData.sh]: updated

**Review by m78233 — DISMISSED** [review]

File: deployment-jvm/local-jvm/performance/createPerformanceData.sh [hunk 2]
```diff
@@ -13,10 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FILE=performance/target/performance-*-exec.jar
+JAR_FILE=performance/target/performance-*-exec.jar
 # Create the perf-test dataset
-if [ -f $FILE ]; then
-  java -jar $FILE --performance.action=create
+if [ -f $JAR_FILE ]; then
+  java -jar -Dspring.profiles.active=create,static $JAR_FILE --performance.action=create
```
Comment (dev930018) [inline deployment-jvm/local-jvm/performance/createPerformanceData.sh:19]:
```suggestion
  java -jar -Dspring.profiles.active=create,static $JAR_FILE
```
I'm pretty sure all the create profile does is set `performance.action=create` anyway

**Review by dev930018 — APPROVED** [review]

**Pull request merged**